install-docker.yml

```diff
@@ -1,26 +1,25 @@
---- 
-- hosts: all
-  remote_user: ubuntu
-  become: yes
-
+---
+- name: Find out which python version is avaible
```

Comment:
Why have two plays to do this python3 workaround ... you could use `pre_tasks` in the main play ... or make a role to do this.